docker/pypi/wmagent-couchdb/couchdb-docker-run.sh

```diff
@@ -76,7 +76,7 @@ groupEntry=$(getent group $thisGroup)
 [[ -d $HOST_MOUNT_DIR/certs ]] || mkdir -p $HOST_MOUNT_DIR/certs || exit $?
 [[ -d $HOST_MOUNT_DIR/admin/couchdb ]] || mkdir -p $HOST_MOUNT_DIR/admin/couchdb || exit $?
 [[ -d $HOST_MOUNT_DIR/srv/couchdb/$COUCH_TAG/config  ]] || mkdir -p $HOST_MOUNT_DIR/srv/couchdb/$COUCH_TAG/config  || exit $?
-[[ -d $HOST_MOUNT_DIR/srv/couchdb/$COUCH_TAG/install/database ]] || mkdir -p $HOST_MOUNT_DIR/srv/couchdb/$COUCH_TAG/install/database || exit $?
```

Comment:
Why did you remove this stricture here. The database location is configured quite early, in the Dockerfile at built time: https://github.com/dmwm/CMSKubernetes/blob/f4a43f8a85bb1be3b59b913b59914f5f66014a34/docker/pypi/wmagent-couchdb/Dockerfile#L38 

If you remove this  directory creation here (even though we mount one level up) it would break the container later.